manifest.json

```diff
@@ -1788,6 +1788,32 @@
                 }
             }
         },
+        "epicEaston197.bspimporter": {
+            "name": "BSPImporter", 
+            "description": "Import bsp files into Neos", 
+            "category": "Asset Importing Tweaks", 
+            "website": "https://github.com/epicEaston197/BSPImporter", 
+            "sourceLocation": "https://github.com/epicEaston197/BSPImporter", 
+            "authors": { 
+                "epicEaston197": {
+                    "url": "https://github.com/epicEaston197" 
+                },
+                "amber": {
+                    "url": "https://github.com/kawaiiamber"
+                }
+            },
+            "versions": { 
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/epicEaston197/BSPImporter/releases/tag/1.0.0", 
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/epicEaston197/BSPImporter/releases/latest/download/BSPImporter.dll", 
```

Comment:
This should probably instead be `https://github.com/epicEaston197/BSPImporter/releases/1.0.0/download/BSPImporter.dll`, since the latest release will change in the future if there are any new releases

Reply:
I hope you don't mind me changing it, as there was no activity for a week I thought I'd get this merged as it was the only real blocker for it.